### DISCUSSION TITLE
Create Client API Key over Application API

### DISCUSSION THREAD

**Discussion by Kaesewuerfel** [pterodactyl/panel#2912] [Support]
Is ist possible to create a client api key for an user over the application api? I didn't found it in the api.
If this is not possible, how could I enter an API token via Mysql? Or how is this created by Pterodactyl?

**Comment by DaneEveritt** [comment]
No. You'd have to dig into the code to see how Pterodactyl is generating them, but I'd recommend not doing that.

**Reply by Kaesewuerfel** [reply]
Hey, thank you for your answer. Where can I find the generating in the code?

**Reply by Kaesewuerfel** [reply]
@DaneEveritt

**Reply by Kaesewuerfel** [reply]
#push

**Reply by Kaesewuerfel** [reply]
#push
